metrics-proxy/src/main/java/ai/vespa/metricsproxy/telegraf/Telegraf.java

```diff
@@ -0,0 +1,77 @@
+// Copyright 2020 Oath Inc. Licensed under the terms of the Apache 2.0 license. See LICENSE in the project root.
+package ai.vespa.metricsproxy.telegraf;
+
+import com.google.inject.Inject;
+import com.yahoo.component.AbstractComponent;
+import com.yahoo.system.execution.ProcessExecutor;
+import org.apache.velocity.Template;
+import org.apache.velocity.VelocityContext;
+import org.apache.velocity.app.VelocityEngine;
+
+import java.io.FileWriter;
+
+import static com.yahoo.yolean.Exceptions.uncheck;
+
+/**
+ * @author olaa
+ */
+public class Telegraf extends AbstractComponent {
+
+    private static final String TELEGRAF_CONFIG_PATH = "/etc/telegraf/telegraf.conf";
+    private static final String TELEGRAF_CONFIG_TEMPLATE_PATH = "src/main/resources/templates/cloudwatch_plugin.vm";
```

Comment:
We probably need to consider these file paths, especially where to read the template path from.